build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -412,6 +412,10 @@ message Action {
   // If true, then the `Action`'s result cannot be cached, and in-flight
   // requests for the same `Action` may not be merged.
   bool do_not_cache = 7;
+
+  // List of required supported [NodeProperty][build.bazel.remote.execution.v2.NodeProperty]
+  // keys. If a property is not recognized the server returns INVALID_ARGUMENT.
```

Comment:
Can we beef out this description a bit? In particular, I think the server should have leeway to interpret these properties how it sees fit. For example, some properties may be applicable to directories but not files (or vice-versa).